mifosng-android/src/main/java/com/mifos/mifosxdroid/online/grouploanaccount/GroupLoanAccountFragment.java

```diff
@@ -324,30 +334,28 @@ public void inflateSubmissionDate() {
         mfDatePicker = MFDatePicker.newInsance(this);
 
         tvSubmittedonDate.setText(MFDatePicker.getDatePickedAsString());
+    }
 
-        tvSubmittedonDate.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                mfDatePicker.show(getActivity().getSupportFragmentManager(), FragmentConstants
-                        .DFRAG_DATE_PICKER);
-            }
-        });
-
+    @SuppressWarnings("unused")
```

Comment:
The methods are not marked unused but I've added the annotation just in case! I've attached an image without the tag here!!
![Uploading Screenshot from 2017-01-27 21-06-15.png…]()